### PR TITLE
add edit event screen

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: ClubManagement/ClubManagement/Activities/Base/CreateOrEditEvent.cs [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+using Android.Gms.Location.Places;
+using ClubManagement.Ultilities;
+using Android.Gms.Location.Places.UI;
+using ClubManagement.Fragments;
+using ClubManagement.Controllers;
+using ClubManagement.Models;
+
+namespace ClubManagement.Activities.Base
+{
+    public class CreateOrEditEvent
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Activities/Base/CreateOrEditEvent.cs]: It need to be an Activity, Fragment, Dialog or something else

File: ClubManagement/ClubManagement/Activities/Base/CreateOrEditEvent.cs [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+
+        public DateTime StartTime
+        {
+            set
+            {
+                startTime = value;
+            }
+        }
+
+        public DateTime EndTime
```
Comment (voquanghoa) [inline ClubManagement/ClubManagement/Activities/Base/CreateOrEditEvent.cs]: Make it 
```C#
public DateTime EndTime{private set;get;}
```

File: ClubManagement/ClubManagement/Activities/Base/CreateOrEditEvent.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply ClubManagement/ClubManagement/Activities/Base/CreateOrEditEvent.cs]: Done

File: ClubManagement/ClubManagement/Activities/Base/CreateOrEditEvent.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply ClubManagement/ClubManagement/Activities/Base/CreateOrEditEvent.cs]: Done

**Pull request merged**